### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by pavel-dudek**
### Refence a related issue

This pull request solves #2

### Description of the changes

- Project now prints sentiment as a number of %

### Mentions

Person responsible for reviewing: @pavel-school

### CODE REVIEW

**Review by pavel-dudek — COMMENTED** [review]
looks ok

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ If applicable, add screenshots to help explain your problem.
 
 **Environment (where are you running the Python Notebook):**
  - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
+ - Browser [e.g. chrome, safari, firefox]
```
Comment (pavel-dudek) [inline .github/ISSUE_TEMPLATE/bug-report.md:28]: ok

Comment (pavel-dudek) [reply .github/ISSUE_TEMPLATE/bug-report.md:28]: looks ook

File: .github/ISSUE_TEMPLATE/feature-request-template.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
-name: Feature request
+name: Feature request template
 about: Suggest an idea for this project
 title: ''
-labels: enhancement
```
Comment (pavel-dudek) [inline .github/ISSUE_TEMPLATE/feature-request-template.md:5]: looks ok

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+.DS_Store
```
Comment (pavel-dudek) [inline .gitignore:1]: looks ok

**Review by pavel-dudek — COMMENTED** [review]
ok

File: .gitignore (hunk 3 above)
Comment (pavel-dudek) [inline .gitignore:1]: ok

**Review by pavel-dudek — COMMENTED** [review]
ok

File: .gitignore (hunk 3 above)
Comment (pavel-dudek) [inline .gitignore:1]: ok

**Review by pavel-dudek — COMMENTED** [review]
ok

**Review by pavel-dudek — COMMENTED** [review]
ok

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ If applicable, add screenshots to help explain your problem.
 
 **Environment (where are you running the Python Notebook):**
  - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
```
Comment (pavel-dudek) [inline .github/ISSUE_TEMPLATE/bug-report.md:28]: ok

**Review by pavel-dudek — COMMENTED** [review]
ok

**Review by pavel-school — APPROVED** [review]
ok, I approve

**Pull request merged**